exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/MariaDBDialect.kt

```diff
@@ -69,5 +76,18 @@ class MariaDBDialect : MysqlDialect() {
         return super.createIndex(index)
     }
 
-    companion object : DialectNameProvider("MariaDB")
+    override fun sequences(): List<String> {
+        val sequences = mutableListOf<String>()
+        TransactionManager.current().exec("SELECT SEQUENCE_NAME FROM INFORMATION_SCHEMA.SEQUENCES") { rs ->
+            while (rs.next()) {
+                sequences.add(rs.getString("SEQUENCE_NAME"))
+            }
+        }
+        return sequences
+    }
```

Comment:
I've tested it again with MariaDB 10.3 and 
````
const val SEQUENCE_MIN_MAJOR_VERSION = 10
const val SEQUENCE_MIN_MINOR_VERSION = 3
````
At least several tests are fail. Especially important is [org.jetbrains.exposed.sql.tests.shared.ddl.SequencesTests#testManuallyCreatedSequenceExists](https://github.com/devgor88/Exposed/blob/cf1c97e3798d8d222b839f296207abef9a42b0ce/exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/SequencesTests.kt#L215)

Overriding can be omitted for versions between 10.3 and 11.5 because it and its super functions return empty list. For versions more or equal 11.5 it can be omitted too because super function is working correctly but I'd left it because super version run this query
````
SELECT TABLE_SCHEMA TABLE_CAT, NULL  TABLE_SCHEM,  TABLE_NAME, IF(TABLE_TYPE='BASE TABLE' or TABLE_TYPE='SYSTEM VERSIONED', 'TABLE', IF(TABLE_TYPE='TEMPORARY', 'LOCAL TEMPORARY', TABLE_TYPE)) as TABLE_TYPE, TABLE_COMMENT REMARKS, NULL TYPE_CAT, NULL TYPE_SCHEM, NULL TYPE_NAME, NULL SELF_REFERENCING_COL_NAME,  NULL REF_GENERATION FROM INFORMATION_SCHEMA.TABLES WHERE  TABLE_TYPE IN ('SEQUENCE') ORDER BY TABLE_TYPE, TABLE_SCHEMA, TABLE_NAME
````

Reply:
What tests are failing for you?

Reply:
I rerun them again and none of sequence related tests are failing.